build.gradle

```diff
@@ -14,6 +14,7 @@ dependencies {
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
     implementation 'io.reactivex.rxjava2:rxjava:2.2.3'
     implementation 'org.jetbrains.kotlin:kotlin-reflect:1.3.10'
+    compile "org.jetbrains.kotlin:kotlin-reflect:1.1.0"
```

Comment:
Please drop this line, we already an `implementation` in place (one line above it).